brigade/plugins/functions/__init__.py

```diff
@@ -0,0 +1,6 @@
+from . import text
```

Comment:
What other types of function plugins do you see?

Was this some thought about making it easier to run things remotely in the future or wouldn't the print_title function just be a helper within Brigade core?

Reply:
A function is something that doesn't apply to hosts themselves but are nice to have. I expect git functions as well, for instance. But effectively a function is a helper as you called it. I just wanted to follow same structure as plugins to make clear that they are not part of the core and they provide extra functionality to users, not to tasks or to the core itself.

Reply:
Ok, sounds good.